### DISCUSSION TITLE
How to freeze columns in data table component

### DISCUSSION THREAD

**Discussion by pedrovgp** [epicmaxco/vuestic-ui#3097] [Q&A] [answered]
Also asked in [Stackoverflow](https://stackoverflow.com/questions/75592185/how-to-freeze-columns-in-vuestic-ui-data-table-vue3).

I am using the [data table component](https://vuestic.dev/en/ui-elements/data-table). I would like to accomplish [this effect](https://hc200ok.github.io/vue3-easy-data-table-doc/features/fixed-column.html) of freezing the first two columns of the table.

Is it possible with the Vuestic data table component and, if so, how can it be achieved?

**Comment by asvae** [comment, marked as answer]
Hey.

We don't support it yet, I moved your feature request to this issue, which you can follow #3099 🤓.

We also have [ag-grid-theme](https://vuestic.dev/en/extensions/ag-grid), which is something you might consider.

**Comment by m0ksem** [comment]
No. We don't have this feature in VaDataTable. But, we recommend to use AgGrid for complex data tables.


Here is what you want:
https://vuestic.dev/en/extensions/ag-grid
https://www.ag-grid.com/vue-data-grid/column-pinning/
